### PR TITLE
INHEART-387: Create bid table

### PR DESCRIPTION
**Pull request by weronikaw29**
I created table with bids and two endpoints: /bids and /bids/create.

### CODE REVIEW

**Review by captain-yossarian — APPROVED** [review]

File: apps/backend/src/app/bids/dto/create-bid.dto.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { IsInt, IsNumber, IsPositive, Max } from 'class-validator';
+
+export class CreateBidDto {
+  @IsPositive()
+  @Max(99999)
```
Comment (pawelroguski) [inline apps/backend/src/app/bids/dto/create-bid.dto.ts:5]: This value would be in env file

File: apps/backend/src/app/bids/bids.service.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export class BidsService {
+  constructor(
+    @InjectRepository(Bid)
+    private bidsRepository: Repository<Bid>
+  ) {}
+  create(createBidDto: CreateBidDto) {
+    const bid = this.bidsRepository.create(createBidDto);
+
+    return this.bidsRepository.save(bid);
+  }
```
Comment (pawelroguski) [inline apps/backend/src/app/bids/bids.service.ts:16]: In my opinion i would add enter here ;)

**Review by pawelroguski — COMMENTED** [review]
Consider suggestions in comments

File: apps/backend/src/app/bids/bids.controller.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { Controller, Get, Post, Body } from '@nestjs/common';
+import { BidsService } from './bids.service';
+import { CreateBidDto } from './dto/create-bid.dto';
+
+@Controller('bids')
+export class BidsController {
+  constructor(private readonly bidsService: BidsService) {}
+
+  @Post('/create')
```
Comment (pawelroguski) [inline apps/backend/src/app/bids/bids.controller.ts:9]: I think that **create** word is not necessary in this place. We adopted standard REST Api convension that methods of sending request decides what happening in application. Post method is always responsible for creating resource ;)

**Pull request merged**